<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideInfoWidget</class>
 <widget class="QWidget" name="SlideInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slide Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="levelDownsampleLabel">
     <property name="text">
      <string>Level, Downsample:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="levelSizeLabel">
     <property name="text">
      <string>Level Size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="mouseSceneLabel">
     <property name="text">
      <string>Mouse Scene:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="viewSceneLabel">
     <property name="text">
      <string>View Scene:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="selectedAreaLabel">
     <property name="text">
      <string>Selected Area (0-level):</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
